modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheConfigurationSplitterImpl.java

```diff
@@ -105,6 +115,27 @@ public CacheConfigurationSplitterImpl(Marshaller marshaller) {
         return new CacheConfigurationEnrichment(enrichment, fieldClsNames);
     }
 
+    /**
+     *  Checks the field on support of splitter.
```

Comment:
Returns {@code true} if this splitter supports the specified field.

Reply:
Done.